packages/test-helpers/package.json

```diff
@@ -28,7 +28,7 @@
   },
   "dependencies": {
     "@open-wc/testing": "^3.0.0-next.5",
```

Comment:
We should check if we can upgrade this package.

Reply:
I overlook on this point. It should be updated to latest version.